app/middleware/validateEmail.js

```diff
@@ -60,16 +61,4 @@ const validateFields = (email, confirmEmail, errors) => {
   return fields;
 };
 
-const setErrorFields = (field, fields, result, errors) => {
-  fields.error = true;
-  fields[field].error = true;
-  fields[field].errorMessage = result.error.message;
-
-  const type = result.error.details[0].type;
-  fields[field].errorHeading = (type === 'any.empty') ?
-    errors.emptyStringHeading : errors.notValidHeading;
```

Comment:
You haven't factored in `emptyStringHeading ` when pulling this function out into it's own file.

**Previously**
![previous](https://user-images.githubusercontent.com/964268/35280334-c7795254-0047-11e8-8115-1f9729fcfd47.png)
**Now**
![now](https://user-images.githubusercontent.com/964268/35280340-cb9e2e7c-0047-11e8-8bcb-19379a8db9aa.png)

Reply:
oops again